### PR TITLE
add requirements_2024.3.txt

### PR DESCRIPTION
**Pull request by wgzintel**

### CODE REVIEW

File: llm_bench/python/requirements_2024.3.txt
Comment (ilya-lavrenov) [inline llm_bench/python/requirements_2024.3.txt:1]: we are already on 2024.3 branch, why do we need such requirements file?

Comment (peterchen-intel) [reply llm_bench/python/requirements_2024.3.txt:1]: @ilya-lavrenov  as part of reproducibility insurance, we add this file to record the versions of dependencies which have been validated by validation team.

Comment (ilya-lavrenov) [reply llm_bench/python/requirements_2024.3.txt:1]: Can we do it directly in requirements.txt file ? 
Once we release 2024.3, we can fix all the dependencies

Comment (peterchen-intel) [reply llm_bench/python/requirements_2024.3.txt:1]: https://github.com/openvinotoolkit/openvino.genai/blob/releases/2024/3/llm_bench/python/requirements.txt  has loser version limitation which allows different 3rd version number.

File: llm_bench/python/requirements_2024.3.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+--extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
+auto_gptq==0.7.1
+pillow==10.4.0
+torch==2.4.0+cpu
+torchvision==0.19.0+cpu
+transformers==4.42.4
+diffusers==0.29.2
+openvino==2024.3.0
+#optimum is in dependency list of optimum-intel
+git+https://github.com/huggingface/optimum-intel.git@439d61f79cf55d5d0b28334f577b6ac3c5ced28f#egg=optimum-intel
```
Comment (eaidova) [inline llm_bench/python/requirements_2024.3.txt:12]: this is not final validation commit of optimum-intel for release (with GPU accuracy bugs), please use pinned in branch commit

Comment (peterchen-intel) [reply llm_bench/python/requirements_2024.3.txt:12]: @eaidova Do you mean this one? https://github.com/openvinotoolkit/openvino.genai/blob/releases/2024/3/llm_bench/python/requirements.txt#L14

Comment (wgzintel) [reply llm_bench/python/requirements_2024.3.txt]: Updated

**Review by peterchen-intel — APPROVED** [review]

**Pull request merged**